src/lib.rs

```diff
@@ -14,64 +14,66 @@ pub mod package;
 /// Supported registries
 pub mod registry;
 
+use std::sync::Arc;
+
 #[cfg(feature = "build")]
 pub use generator::Language;
 
+use credentials::Credentials;
+use eyre::{ContextCompat, WrapErr};
+use manifest::Manifest;
+use package::PackageStore;
+use registry::Artifactory;
+
+pub const JFROG_AUTH_HEADER: &str = "X-JFrog-Art-Api";
```

Comment:
can this be local to the artifactory registry impl + private? this is not useful for anyone using the rust lib

Reply:
Yeah, good point. I moved it as close to the code that consumes it as possible.